File: 100-Days-Series/Day 87/PerfectNumber.cpp
#include<iostream>
using namespace std;
// Function to print Perfect Square Number between 1 to N
void getPrintPerfectSquareNumber(int n){
    for(int i=1;i*i<=n;i++){
        cout<<(i*i)<<"  ";
    }
}
// Let's Start Here
int main(){
    int n;
    cout<<"\nEnter the value of N : ";
    cin>>n;
    cout<<"\nPerfect Square Number between 1 to "<<n<<" :"<<endl;
    getPrintPerfectSquareNumber(n);
    return 0;
}

File: 100-Days-Series/Day 87/PerfectNumber.py

# Function to print perfect number Square between 1 to N
def get_print_perfect_square_number(n):
    for i in range(1,n+1,1):
        res=i*i
        if(res>n):
            break
        print(res,end=" ")

# Let's Start Here
n=int(input('Enter the value of N : '))
print('Perfect Square Number between 1 and',n,' :')
get_print_perfect_square_number(n)